TypeProvider/Core/Naming.fs

```diff
@@ -0,0 +1,30 @@
+﻿[<RequireQualifiedAccess>]
+module internal Froto.TypeProvider.Core.Naming
```

Comment:
Not sure I follow you about mapping protobuf scalar types -> .NET types; seems like these should be fixed: e.g., int32 -> int32.

Do you have an example use case?  Can you explain a bit further?